docs/ref/django-admin.txt

```diff
@@ -1828,6 +1828,8 @@ Displays a full stack trace when a :exc:`~django.core.management.CommandError`
 is raised. By default, ``django-admin`` will show an error message when a
 ``CommandError`` occurs and a full stack trace for any other exception.
 
+This option is not supported in ``manage.py runserver``.
+
```

Comment:
I've added it only for consistency with other options for django-admin: 'settings' and 'pythonpath', where comment reads:
> This option is unnecessary in ``manage.py``, because...

Are you sure we should not mention it for runserver?

Reply:
OK, let's leave it. I'd change to `ignored`:
```suggestion
This option is ignored by :djadmin:`runserver`.

```